module-4/backend/week_one.md

```diff
@@ -4,18 +4,63 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
+
 1. What's the most useful thing you learned from completing the intermission week work?
+
+I enjoyed doing Sorting Suite. I got a lot more comfortable with Javascript/Chai getting to play with it in a familiar context. It helped me to understand a lot of the similarities between Javascript and Ruby, which helps to isolate the differences.
+
 2. What are some tools to help debug JavaScript code?
+
+Debugger is my favorite. There's also console.log and pry
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+We've used Chai so far and are just getting started using Selenium. I'm not comfortabe with Selenium yet, but hopefully I will be by the time I get through Quantified Self.
+
 4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. What's a callback function and what are some reasons when we use/need callback functions?
-8. What's the biggest difference between a promise and a callback?
-9. How do we setup a route when creating an API with Node and Express?
-10. What's `npm` and what do we use it for?
+
+myFunction()
+
+5. What is CORS?
```

Comment:
Definitely look into this! It's pretty straightforward and a good thing to know about.